specification/paths/MultiColliShipments.json

```diff
@@ -19,9 +19,17 @@
           "additionalProperties": false,
           "properties": {
             "data": {
-              "type": "array",
-              "items": {
-                "$ref": "#/definitions/Shipment"
+              "type": "object",
+              "properties": {
+                "shipment": {
```

Comment:
I'd be okay with calling this `shipment` but maybe `master_shipment` would be more descriptive. How do we make the distinction in our database/API?

Reply:
In our database it's a normal shipment record. It has a property `is_multi_colli = true` to easily filter them from the response, but in practice it's a regular shipment (with other shipments linked to it).

Another way is to introduce a `colli` property on the shipment attributes (just like the items, which are attributes here, but relationships in our API).